### DISCUSSION TITLE
Welcome to MIDI2LR Discussions!

### DISCUSSION THREAD

**Discussion by rsjaffe** [rsjaffe/MIDI2LR#516] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hdstead** [comment]
Active photographer using LrC with MIDI2LD.

**Comment by Enricopez** [comment]
Hello everyone, wedding and life style photographer using Midi2lr every day for spinning my workflow.

**Comment by JKCothron** [comment]
Nice change!  I have been using Midi2LR for quite a while now.  I'm a serious hobbyist, primarily landscape photography.

**Comment by Enricopez** [comment]
Make midi2LR for iPad 🥰
